docs/index.rst

```diff
@@ -122,6 +122,11 @@ ar71xx-generic
 
   - Carambola 2
 
+* Aerohive
+
+  - HiveAP 121
+  - HiveAP 330
+
```

Comment:
They're not in ar71xx-generic, are they?

Reply:
Sorry, fixed that. They are in their respective target now